src/tools/perf/api/libperf.h

```diff
@@ -86,7 +86,8 @@ enum ucx_perf_test_flags {
     UCX_PERF_TEST_FLAG_STREAM_RECV_DATA = UCS_BIT(8), /* For stream tests, use recv data API */
     UCX_PERF_TEST_FLAG_FLUSH_EP         = UCS_BIT(9), /* Issue flush on endpoint instead of worker */
     UCX_PERF_TEST_FLAG_WAKEUP           = UCS_BIT(10), /* Create context with wakeup feature enabled */
-    UCX_PERF_TEST_FLAG_ERR_HANDLING     = UCS_BIT(11) /* Create UCP eps with error handling support */
+    UCX_PERF_TEST_FLAG_ERR_HANDLING     = UCS_BIT(11), /* Create UCP eps with error handling support */
+    UCX_PERF_TEST_FLAG_LOOPBACK         = UCS_BIT(12)  /* Use loopback transport */
```

Comment:
Maybe don't use the word "transport" here (in the comment)? Because loopback is not a transport type, but rather a "mode".
WDYT of something like:
```
/* Use loopback  */
```

or :
```
/* Run in loopback mode */
```
?